assets/icons/security.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="20" height="24" viewBox="0 0 20 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can you adjust the two icons such that they are stylable via our icon component please? i.e. setting `fill="currentColor"`? Same for `support.svg`. Nearly all of our other icons have a size of 24x24 px.